notebooks/Tutorial3_Embedded_processors.ipynb

```diff
@@ -181,7 +181,7 @@
    "id": "cc128b4d",
    "metadata": {},
    "source": [
-    "We observe that we the avatarization keeps the same proportion of missing values. But the location of the missing value cells in the data is not similar. This is because the missing value characteristics has also been anonymized."
+    "We observe that the avatarization keeps aproximatively the same proportion of missing values. But the location of the missing value cells in the dataset is not similar. This is because the missing value characteristics has also been anonymized."
```

Comment:
```suggestion
    "We observe that the avatarization keeps approximately the same proportion of missing values. But the location of the missing value cells in the dataset is not similar. This is because the missing value characteristics has also been anonymized."
```